pkg/ocmagenthandler/ocmagenthandler.go

```diff
@@ -68,6 +68,7 @@ func (o *ocmAgentHandler) EnsureOCMAgentResourcesExist(ocmAgent ocmagentv1alpha1
 		ensureSecretFunc,
 		o.ensureService,
 		o.ensureNetworkPolicy,
+		o.ensureNetworkPolicyForMUO,
```

Comment:
Can we do the same as Configmap above? We call the `o.ensureNetworkPolicy` or `o.ensureAllNetworkPolicies` once here. And proceed all the required networkpolicies within networkpolicy.go

Reply:
> Can we do the same as Configmap above? We call the `o.ensureNetworkPolicy` or `o.ensureAllNetworkPolicies` once here. And proceed all the required networkpolicies within networkpolicy.go

That's a good suggestion, I will try to refactor to align with.